frontend/components/Dashboard/Profile/PostList.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { useTranslation } from '../../../config/i18n'
+import { layoutLeftRightPadding } from '../../Dashboard/dashboardConstants'
+import ProfilePostCard from '../PostCard/ProfilePostCard'
+import theme from '../../../theme'
+
+type Props = {
+  // TODO: use Post type from generated graphql
```

Comment:
This may be fixed now since we've improved all the backend stuff a ton. It's worth giving it a try since it came up in this PR a few times